File: prm392_assignment/MyApplication/app/src/main/java/com/example/myapplication/model/req/AddExpenseRequest.java
package com.example.myapplication.model.req;

public class AddExpenseRequest {
    private int categoryId;
    private double amount;
    private String date;
    private String description;

    public AddExpenseRequest(int categoryId, double amount, String date, String description) {
        this.categoryId = categoryId;
        this.amount = amount;
        this.date = date;
        this.description = description;
    }
}

File: prm392_assignment/PRM392_ExpenseManagement/ExpenseMangamentAPI/DTO/Users/AddExpenseRequest.cs
namespace ExpenseMangamentAPI.DTO.Users
{
    public class AddExpenseRequest
    {
        public int CategoryId { get; set; }
        public decimal Amount { get; set; }
        public DateTime? Date { get; set; } // Ngày có thể là null, mặc định là ngày hiện tại
        public string? Description { get; set; }
    }
}
